### ISSUE TITLE
Language: Add tag on bears: Localbear or Globalbear

### ISSUE THREAD

**Issue by khanchi97** [coala/coala-bears#2119]
One should be able to search `localbears` or `globalbears` on this page https://coala.io/#/languages

**Comment by khanchi97** [comment]
@Makman2 what do you say about this?

**Comment by Makman2** [comment]
Not sure this is actually useful. If you want to lint something, you don't care whether it's a local or global bear, it's imo implementation detail (it only controls the parallelization).

**Comment by khanchi97** [comment]
Maybe it would be helpful for coala-developers for some queries.

**Comment by Makman2** [comment]
Hmm maybe... However as a developer I would always look into the relevant .py file where the bear resides, then I've everything what I need.

I don't want to turn down the idea, i just want to discuss it if it's really worth it and useful, and also not confusing :) for example instead of this local/global bear info only, we maybe want to extend this idea to some nice developer interface for everything bear related (source view, stats, executable using - only relevant for linter bears, .....)

**Comment by khanchi97** [comment]
>we maybe want to extend this idea to some nice developer interface for everything bear related (source view, stats, executable using - only relevant for linter bears, .....)

Yeah, it's a good idea.Let see what will be sorted out from this discussion thread.IMO we should implement this.

**Comment by Makman2** [comment]
So generally I have no problem with some online dev info, however for me it wouldn't worth it to implement some fancy website stuff, as that's what GitHub's for (at least for me). Every information is inside the relevant Python file, and if you can read Python you usually don't have a problem (+ the bear guides in this case).

However, some ideas anyway what could be presented as info for a bear:
- Local or Global Bear?
- Is linter bear
  - If linter bear:
    - executable
    - Other flags? (`output_format`, `use_stdout`, ...)
